chia/cmds/farm_funcs.py

```diff
@@ -132,10 +133,15 @@ def process_harvesters(harvester_peers_in: Dict[str, Dict[str, Any]]) -> None:
                     print(f"   Loading plots: {syncing['plot_files_processed']} / {syncing['plot_files_total']}")
                 else:
                     total_plot_size_harvester = harvester_dict["total_plot_size"]
+                    total_effective_plot_size_harvester = harvester_dict["total_effective_plot_size"]
                     plot_count_harvester = harvester_dict["plots"]
                     PlotStats.total_plot_size += total_plot_size_harvester
+                    PlotStats.total_effective_plot_size += total_effective_plot_size_harvester
                     PlotStats.total_plots += plot_count_harvester
-                    print(f"   {plot_count_harvester} plots of size: {format_bytes(total_plot_size_harvester)}")
+                    print(
+                        f"   {plot_count_harvester} plots of size: {format_bytes(total_plot_size_harvester)} raw, "
```

Comment:
I think "on-disk" is better than "raw". that's what it means, right?

Reply:
Thanks! I'll change the text.